File: grind75/5-valid-palindrome.js
let isPalindrome = function (s) {
  let string = []
  for (let i = 0; i < s.length; i++) {
    let char = s[i].toLowerCase()
    let alphanumeric = 'abcdefghijklmnopqrstuvwxyz0123456789'
    if (alphanumeric.includes(char)) {
      string.push(char)
    }
  }
  let reversed = [...string].reverse()
  return string.join('') === reversed.join('')
};

File: grind75/5-valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        # create a string, add if character is alphanumeric
        # create a reverseed string
        # check if the strings are equal
        s = s.casefold()
        string = ''
        for char in s:
            if char.isalnum():
                string += char

        reverse = string[::-1]

        return string == reverse